dcp_prototype/backend/wrangling/migrations/scripts/README.md

```diff
@@ -0,0 +1,56 @@
+# Scripts to copy data from DCP 1.0
+
+### migrations/scripts/get_project_metadata.py
+
+
+This script will download bundles, bundle_manifests and files and store them where the function was called for all of 
+the projects in the project_list
+```bash
+├── hold_data
+│   ├── ProjectShortName/
+│   │   ├── bundle_manifests/
+│   │   ├── bundles/
+│   │   ├── errors
+│   │   └── files/
+```
+Example use
+```bash
+python3 ./dcp_prototype/backend/wrangling/migrations/scripts/get_project_metadata.py -p <project_list> --threads <nthreads>
+```
+Concrete use
+```bash
+python3 ./dcp_prototype/backend/wrangling/migrations/scripts/get_project_metadata.py -p ./dcp_prototype/backend/wrangling/migrations/projects.json --threads 20
+```
+If --thread not passed in, defaults to 1
+
+### migrations/scripts/load_project_metadata.py
+
+This script will load all files (without repeats within a project) and will rename files to {uuid}.{version}.json (with exception of links.json which will be renamed to its checksum)
```

Comment:
What exactly do you mean by "will load all files"?

Reply:
updated